addons/mail/models/ir_actions.py

```diff
@@ -51,6 +58,29 @@ def run_action_email(self, action, eval_context=None):
         action.template_id.send_mail(self._context.get('active_id'), force_send=False, raise_exception=False)
         return False
 
+    @api.model
+    def run_action_next_activity(self, action, eval_context=None):
```

Comment:
Why don't you use the ``activity_schedule`` method defined on the mail module ?

Reply:
Because as it's a generic implementation I can't use the mixin (and then this method): I don't know on which model it will apply on.

Am I wrong?